### PR TITLE
should not ignore return messages from wait function

### PR DESCRIPTION
**Pull request by stewart-yu**
**What this PR does / why we need it**:
It should not ignore return messages for `wait*` function. When it go wrong, need `return` at once.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/cloudprovider/providers/azure/azure_blobDiskController.go [hunk 1]
```diff
@@ -519,7 +519,7 @@ func (c *BlobDiskController) createStorageAccount(storageAccountName string, sto
 			return fmt.Errorf("azureDisk - can not create new storage account, current storage accounts count:%v Max is:%v", len(c.accounts), maxStorageAccounts)
 		}
 
-		glog.V(2).Infof("azureDisk - Creating storage account %s type %s \n", storageAccountName, string(storageAccountType))
+		glog.V(2).Infof("azureDisk - Creating storage account %s type %s\n", storageAccountName, string(storageAccountType))
```
Comment (andyzhangx) [inline pkg/cloudprovider/providers/azure/azure_blobDiskController.go]: what has this PR to do with `azure_blobDiskController.go`? only delete a space? It's ok but not related to this PR...

Comment (stewart-yu) [reply pkg/cloudprovider/providers/azure/azure_blobDiskController.go]: just follow [https://github.com/kubernetes/kubernetes/pull/56409](https://github.com/kubernetes/kubernetes/pull/56409), i see also in  `/provider` directory, just fix it.

Comment (stewart-yu) [reply pkg/cloudprovider/providers/azure/azure_blobDiskController.go]: Thanks,  i will remove it

**Comment by dims** [comment]
/holde

looks like there's a TODO on this

**Comment by stewart-yu** [comment]
Can you provide more information about it? may be i can track about it @dims Thanks

**Comment by dims** [comment]
/hold cancel

oops. looks like you already took care of it you had mentioned "Thanks, i will remove it"

**Pull request merged**